stable/aws-es-proxy/values.yaml

```diff
@@ -0,0 +1,67 @@
+# aws-es-proxy specific coniguration
+config:
+  # awsSecretAccessKey: "XXXXXXX"
+  # awsAccessKeyID: "XXXXXXXX"
+  endpoint: https://test-es-somerandomvalue.eu-west-1.es.amazonaws.com
+  pretty: false
+  verbose: false
+
+image:
+  repository: "abutaha/aws-es-proxy"
+  tag: "0.8"
+  pullPolicy: "IfNotPresent"
+
+# Optionally specify an array of imagePullSecrets.
+# Secrets must be manually created in the namespace.
+# ref: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+# imagePullSecrets:
+  # - name: myRegistryKeySecretName
+
+extraArgs: {}
+  # -log-to-file
+  # -no-sign-reqs
+
+service:
+  type: ClusterIP
+  port: 9200
+  annotations: {}
+  # foo.io/bar: "true"
```

Comment:
why this?

Reply:
As an example to make sure the user gets how to configure annotations. But I'm fine with removing it.